Source/WebCore/html/track/TextTrackCue.cpp

```diff
@@ -65,13 +65,13 @@ namespace WebCore {
 WTF_MAKE_ISO_ALLOCATED_IMPL(TextTrackCue);
 WTF_MAKE_ISO_ALLOCATED_IMPL(TextTrackCueBox);
 
-static const QualifiedName& cueAttributName()
+static const QualifiedName& cueAttributeName()
 {
     static NeverDestroyed<QualifiedName> cueTag(nullAtom(), "cue"_s, nullAtom());
     return cueTag;
 }
 
-static const QualifiedName& cueBackgroundAttributName()
+static const QualifiedName& cueBackgroundAttributeName()
 {
     static NeverDestroyed<QualifiedName> cueBackgroundTag(nullAtom(), "cuebackground"_s, nullAtom());
     return cueBackgroundTag;
```

Comment:
Should we instead add these to `Source/WebCore/html/HTMLAttributeNames.in`? Presumably we need IDL attributes for them as well and such?

Reply:
I think you should be able to get rid of these now that you added them there.